### PR TITLE
fix: remove 'local' under storage class parameters

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

**Review by GlennBullingham — COMMENTED** [review]
As discussed on our earlier call, I think we should take this out but include an Info panel which calls out that this parameter has been deprecated as of this version.

**Review by avishnu — APPROVED** [review]
lgtm

File: reference/storage-class-parameters.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Storage class resource in Kubernetes is used to supply parameters to volumes when they are created. It is a convenient way of grouping volumes with common characteristics. All parameters take a string value. Brief explanation of each supported Mayastor parameter follows.
 
+
+{% hint style="info" %}
+The storage class parameter `local` has been deprecated in Mayastor version 2.0.
```
Comment (GlennBullingham) [inline reference/storage-class-parameters.md]: Sorry to have not raised it earlier during your edit cycle but I'm only just thinking that we should probably call this out explicitly as a breaking change, and make it clear that it should not be used.

**Review by GlennBullingham — APPROVED** [review]
LGTM, with one comment regarding the nature of the 'local' parameter change in the storageclass, which I think we should address.

**Pull request merged**